release/models/bgp/openconfig-bgp-types.yang

```diff
@@ -713,23 +722,24 @@ module openconfig-bgp-types {
   typedef community-type {
     type enumeration {
       enum STANDARD {
-        description "Send only standard communities";
+        description "Send standard communities";
       }
       enum EXTENDED {
-        description "Send only extended communities";
+        description "Send extended communities";
       }
-      enum BOTH {
-        description "Send both standard and extended communities";
+      enum LARGE {
+        description "Send large communities";
       }
-      enum NONE {
```

Comment:
You will need to leave NONE as an enum to keep this a non-breaking change.  

At https://github.com/openconfig/public/pull/852/files#diff-d8b66cec430d0c380b54bb8473f2dfe90c54e898a338c53c0d321e0c6fae1ac3R289 you should set the  default to be NONE .